examples/basic-bidding-sale/basic-bidding-sale.repl

```diff
@@ -190,23 +214,43 @@
 
   (env-data {
     "bid-id": (get-bid-id "DKc5HEWcmP8iPWue2WvMcnjn_WnowmuHz5Ogukm2SWk" "k:bidder")
-    ,"market-guard": {"keys": ["market"], "pred": "keys-all"}
-    ,"seller-guard": {"keys": ["account"], "pred": "keys-all"}
     ,"sale-id": "DKc5HEWcmP8iPWue2WvMcnjn_WnowmuHz5Ogukm2SWk"
   })
 
-  (marmalade-v2.abc.create-account "k:market" (read-keyset 'market-guard))
-
   (env-sigs
-  [{'key:'account
-    ,'caps: [
-      (UPDATE_QUOTE_GUARD (read-msg 'sale-id))
-      (SELLER (read-msg 'sale-id))
-    ]}
+    [{'key:'account
```

Comment:
Should we make separate caps for `offer-for-auction`, and `accept-bid`  instead of re-using `SELLER` cap?

Reply:
I personally think in this case the `SELLER` cap is more descriptive and thus makes the code easier to understand. I don't think it's wrong to re-use a cap in essence, and I feel in this case it's justified, since we're not guarding calls to external modules for example. 
Do you see another reason why we shouldn't re-use the cap in this case?

Reply:
By separating the caps, we can let the seller know what they're exactly signing for, especially this is a two separate activities. I think we can reuse the seller guard enforcement code, but would be more clear flow to the sellers when signing.

Reply:
Yes, good point. I've updated the code